0_intro/stage/exos_python.ipynb

```diff
@@ -29,6 +29,37 @@
     "5. Écrire une fonction vérifiant que tous les éléments d'une liste sont pairs."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        return [ i  for i in range(2,n) if premier(i) ]
</pre><p>range(n+1) si n est inclus (comme dans l'énoncé)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>